cms/envs/aws.py

```diff
@@ -496,4 +496,4 @@
 
 ############## Settings for Studio Context Sensitive Help ##############
 
-DOC_LINK_BASE_URL = ENV_TOKENS.get('DOC_LINK_BASE_URL', DOC_LINK_BASE_URL)
+HELP_TOKENS_BOOKS = ENV_TOKENS.get('HELP_TOKENS_BOOKS', HELP_TOKENS_BOOKS)
```

Comment:
ugh, bad `git reset origin/master` here. Ignore all changes to `*/envs/*.py`, I was able to get the new backend working *only* for tasks that opt-in to it.